storage/range.go

```diff
@@ -124,17 +124,16 @@ var configDescriptors = [...]*configDescriptor{
 
 // tsCacheMethods specifies the set of methods which affect the
 // timestamp cache.
-var tsCacheMethods = [...]bool{
-	proto.Contains:                   true,
-	proto.Get:                        true,
-	proto.Put:                        true,
-	proto.ConditionalPut:             true,
-	proto.Increment:                  true,
-	proto.Scan:                       true,
-	proto.Delete:                     true,
-	proto.DeleteRange:                true,
-	proto.InternalResolveIntent:      true,
-	proto.InternalResolveIntentRange: true,
+var tsCacheMethods = [...]struct{}{
```

Comment:
Oh, I see now what's happening here (and why CircleCI fails). This is a slice and you need the bool. Either change this to a map if you want the below to work, or keep it as a slice, but then you need the bool.